src/Controls/src/Core/HandlerImpl/Window.Impl.cs

```diff
@@ -94,13 +100,26 @@ void OnCollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
 				ModalNavigationService.SettingNewPage();
 
 				if (value != null)
+				{
 					value.AttachedHandler += OnPageAttachedHandler;
+					value.DetachedHandler += OnPageDetachedHandler;
+				}
 			}
 		}
 
 		void OnPageAttachedHandler(object? sender, EventArgs e)
 		{
 			ModalNavigationService.PageAttachedHandler();
+#if !NETSTANDARD
+			AlertManager.Subscribe();
```

Comment:
Can we make the netstandard calls on these do nothing so we can get rid if these if/def checks?

Reply:
Sure, changed.